Dockerfile

```diff
@@ -26,26 +39,25 @@ RUN yum -y upgrade && yum -y install \
     python3-devel \
     python3-setuptools \ 
     readline-devel \
+    sudo \
     unzip \ 
     xorg-x11-server-Xvfb \
     xorg-x11-utils \
     xz \
     zlib-devel
 
+RUN dnf --enablerepo=devel install sed
 
 RUN yum -y group install "Development Tools"
 
-# Get fakeroot which needs epel-release 
```

Comment:
Is there a reason you've removed the comments, I think they help explain the different steps of the dockerfile